experiment/procedure.md

```diff
@@ -1 +1,13 @@
-### Procedure
\ No newline at end of file
+ 
+**Procedure**
+
+1. Dimensions of test piece is measured at 3 different places along its height/length to determine the average c/s area.
+2. Ends of the specimen should be plane. For that the ends are tested on a bearing plate.
```

Comment:
The spelling of plane here is different from the spelling in posttest question 1 (plain). Please use the correct spelling at both places.